docker/Dockerfile-carla-mot

```diff
@@ -12,24 +12,4 @@ RUN pip install --no-cache-dir \
 RUN pip install --no-cache-dir \
     cython-bbox
 
-
-WORKDIR /armory-repo
```

Comment:
I'm trying to prove to myself that we will not need this again on the next release build. Please confirm that this only existed because twosixarmory/pytorch:0.15.4 did not have the 0.16 versioning code in it. 

Now that `twosixarmory/pytorch:0.16.0` exists on dockerhub, this will not be needed for a build of `twosixarmory/carla-mot:0.16.1`. Correct?